mscore/scoreview.cpp

```diff
@@ -5958,14 +5965,19 @@ static bool elementLower(const Element* e1, const Element* e2)
                         }
                   }
             }
-      return e1->z() < e2->z();
+      // force z of stem to be seen as higher than note
+      else if (e1->type() == Element::Type::STEM)
+            z1 = n.z() + 1;
+      else if (e2->type() == Element::Type::STEM)
```

Comment:
Sorry I wasn't clear.  I just tested out his PR today, but rebased it ontop of latest master.